app/workers/scheduler/statuses_count_writeback_scheduler.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'sidekiq-scheduler'
+
+class Scheduler::StatusesCountWritebackScheduler
+  include Sidekiq::Worker
+
+  def perform
+    redis.sscan_each(Account::STATUSES_COUNT_UPDATED_CACHE_KEY) do |account_id|
+      StatusesCountWritebackWorker.perform_async(account_id)
+      redis.srem(Account::STATUSES_COUNT_UPDATED_CACHE_KEY, account_id)
```

Comment:
You may move swap this line with the previous line. Otherwise it would have a race condition like this:
1. The scheduler found cache key
2. The scheduler performs status count writeback
3. Another thread performs an update and sets cache key
4. The scheduler removes cache key

In this case, the update performed in step 3 will be ignored.

It is unlikely to be a problem because `StatusesCountWritebackWorker` is asynchronously executed and therefore it would run after the cache key removal in most cases. However I don't think you have to introduce such a danger.